build/build-buildimages.sh

```diff
@@ -81,7 +81,13 @@ touch $ACR_BUILD_IMAGES_ARTIFACTS_FILE
 > $ACR_BUILD_IMAGES_ARTIFACTS_FILE
 
 # Build buildpack images
-source $REPO_DIR/build/build-buildpacks-images.sh
+# 'pack' is not supported on Windows
+if [[ "$OSTYPE" == "linux-gnu" ]] || [[ "$OSTYPE" == "darwin"* ]]; then
+	source $REPO_DIR/build/build-buildpacks-images.sh
+else
+	echo
+	echo "Skipping building 'build packs' as this platform '$OSTYPE' is not supported."
```

Comment:
well then, if we add an OS gate to `build-buildimages.sh` -
shouldn't we remove the existing OS gate from `install-pack.sh`?
And only leave the `Could not detect compatible pack binary platform` condition?